default/stringtables/en.txt

```diff
@@ -1077,6 +1077,9 @@ Print version and exit.
 OPTIONS_DB_SINGLEPLAYER
 Start server in single player host mode. This only allows clients on localhost to connect.
 
+OPTIONS_DB_HOSTLESS
+Start server in hostless mode. Server doesn't require host.
```

Comment:
Should elaborate on this... particularly that it will make the server revert to MPLobby after a game, rather than exiting and requiring a restart.